scripts/msct_image.py

```diff
@@ -237,13 +264,79 @@ def range(self, axis):
         constructing a Slicer on "IS" (or "SI"),
         getting a range on "IS" would get [0,1,2],
         getting a range on "SI" would get [2,1,0].
+
+        Notes:
+
+        - To be used with direct image indexing, not as indexes for Slice[]
+          which are "logical" according to the slicing direction..
+
         """
         if axis == self.axis:
             return range(0, self.nb_slices)
         if axis == self.axis[::-1]:
             return range(self.nb_slices-1, -1, -1)
         raise ValueError()
 
+    def __len__(self):
+       return self.nb_slices
+
+    def __getitem__(self, idx):
+       """
+       :return: an image slice, at slicing index idx
+       :param idx: slicing index (according to the slicing direction)
+       """
+       if isinstance(idx, slice):
+           raise NotImplementedError()
+
+       if idx >= self.nb_slices:
+           raise IndexError("I just have {} slices!".format(self.nb_slices))
+
+       if self.direction == -1:
+           idx = self.nb_slices - 1 - idx
+
+       return self.im.data[self.slice(idx)]
+
+    def __call__(self):
+        """
+        Slice generator
+
+        Example: [slice for slice in Slicer(im)()]
+        """
+        for idx_slice in self.range(self.axis):
+            yield self.im[self.slice(idx_slice)]
+
+
+class SlicerMany(object):
+    """
+    Images slicer utility class.
```

Comment:
this is the same header as for `SlicerSingle`. Would it be possible to explicit a bit more what is the specificity of this function vs. `SlicerSingle` (maybe add an example under each header?).

Reply:
There is an 's' at Images, but you're right. I'll add more docs later. For now this was just added for the cool factor. There's a lot of pre-conditions to document (caveats) for using that `SlicerMany` object.